lib/chef/audit/audit_reporter.rb

```diff
@@ -138,9 +138,13 @@ def post_auditing_data
               return
             else
               # Save the audit report to local disk
-              error_file = "failed-audit-data.json"
+              error_file = "audit-data.json"
```

Comment:
Right, the "api" would change, but the file name makes me think the audit failed, instead of failed to post. I'll defer to you if this should be changed back, but I think it's a change for the better.